website/content/en/v0.10.0/getting-started/migrating-from-cas/_index.md

```diff
@@ -93,6 +93,12 @@ First set the Karpenter release you want to deploy.
 export KARPENTER_VERSION=v0.10.0
 ```
 
+Make sure repo is added to Helm by using below command.
```

Comment:
Nit:  Can we reword this?  

Perhaps something like:  
"Make sure the Karpenter repo is added to Helm by running the following commands"

Reply:
That makes sense. Committed the change.